test/Data/Abstract/Path/Spec.hs

```diff
@@ -7,14 +7,14 @@ spec :: Spec
 spec = parallel $
   describe "joinPaths" $ do
     it "joins empty paths" $
-      joinPaths "" "" `shouldBe` "."
+      joinPathf "" "" `shouldBe` "."
```

Comment:
I will implement the test of joinPaths in another PR along with joinPaths rewrite.